diff -durN -x '*~' nspr-4.4.1.orig/mozilla/nsprpub/config/nspr-config.in nspr-4.4.1/mozilla/nsprpub/config/nspr-config.in
--- nspr-4.4.1.orig/mozilla/nsprpub/config/nspr-config.in	2003-02-28 02:48:15.000000000 +0100
+++ nspr-4.4.1/mozilla/nsprpub/config/nspr-config.in	2005-01-01 00:50:43.057779416 +0100
@@ -2,6 +2,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+libdir=@libdir@
 exec_prefix_set=no
 
 major_version=@MOD_MAJOR_VERSION@
@@ -95,7 +96,7 @@
 fi
 
 if test "$echo_libs" = "yes"; then
-      libdirs=-L${exec_prefix}/lib
+      libdirs=-L${libdir}
       if test -n "$lib_plds"; then
 	libdirs="$libdirs -lplds${major_version}"
       fi
