--- nspr-4.2/mozilla/nsprpub/pr/src/pthreads/ptio.c.orig	Mon Sep  2 14:59:47 2002
+++ nspr-4.2/mozilla/nsprpub/pr/src/pthreads/ptio.c	Mon Sep  2 15:17:32 2002
@@ -55,6 +55,7 @@
 #include <sys/uio.h>
 #include <sys/file.h>
 #include <sys/ioctl.h>
+#include <netinet/tcp.h>
 #if defined(SOLARIS) || defined(UNIXWARE)
 #include <sys/filio.h>  /* to pick up FIONREAD */
 #endif
@@ -291,8 +292,7 @@
  * most current systems.
  */
 #if defined(HAVE_SOCKLEN_T) \
-    || (defined(LINUX) && defined(__GLIBC__) && __GLIBC__ >= 2 \
-    && !defined(__alpha))
+    || (defined(LINUX) && defined(__GLIBC__) && __GLIBC__ >= 2 )
 typedef socklen_t pt_SockLen;
 #elif (defined(AIX) && !defined(AIX4_1)) \
     || (defined(LINUX) && defined(__alpha)) \
