--- nspr-4.10/nspr/configure.in.orig	2013-05-28 23:39:29.000000000 +0200
+++ nspr-4.10/nspr/configure.in	2013-06-08 06:25:10.860289567 +0200
@@ -3193,7 +3193,7 @@
 dnl ========================================================
 dnl Generate output files.
 dnl ========================================================
-MAKEFILES="
+m4_define([MAKEFILES], [
     Makefile
     config/Makefile
     config/autoconf.mk
@@ -3224,41 +3224,17 @@
     pr/src/threads/Makefile
     pr/tests/Makefile
     pr/tests/dll/Makefile
-"
-
-if test "$OS_TARGET" = "Linux"; then
-    MAKEFILES="$MAKEFILES
         pkg/linux/Makefile
-    "
-elif test "$OS_TARGET" = "SunOS"; then
-    MAKEFILES="$MAKEFILES
         pkg/solaris/Makefile
         pkg/solaris/SUNWpr/Makefile
         pkg/solaris/SUNWprd/Makefile
-    "
-fi
-
-if test -z "$USE_PTHREADS" && test -z "$USE_BTHREADS"; then
-    MAKEFILES="$MAKEFILES
         pr/src/threads/combined/Makefile
-    "
-elif test -n "$USE_PTHREADS"; then
-    MAKEFILES="$MAKEFILES
         pr/src/pthreads/Makefile
-    "
-elif test -n "$USE_BTHREADS"; then
-    MAKEFILES="$MAKEFILES
         pr/src/bthreads/Makefile
-    "
-fi
-
-if test -n "$USE_CPLUS"; then
-    MAKEFILES="$MAKEFILES
         pr/src/cplus/Makefile
         pr/src/cplus/tests/Makefile
-    "
-fi
+])
 
-echo $MAKEFILES > unallmakefiles
+echo "MAKEFILES" > unallmakefiles
 
-AC_OUTPUT([$MAKEFILES], [chmod +x config/nspr-config])
+AC_OUTPUT(MAKEFILES, [chmod +x config/nspr-config])
