--- nspr-4.4.1/mozilla/nsprpub/config/nspr.m4.orig	2001-05-12 02:36:57.000000000 +0200
+++ nspr-4.4.1/mozilla/nsprpub/config/nspr.m4	2003-12-29 02:43:51.525951528 +0100
@@ -5,7 +5,7 @@
 
 dnl AM_PATH_NSPR([MINIMUM-VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND]]])
 dnl Test for NSPR, and define NSPR_CFLAGS and NSPR_LIBS
-AC_DEFUN(AM_PATH_NSPR,
+AC_DEFUN([AM_PATH_NSPR],
 [dnl
 
 AC_ARG_WITH(nspr-prefix,
